css.js

```diff
@@ -39,10 +39,24 @@ if (typeof window !== 'undefined') {
             resolve('');
         }, 7);
       };
-      var link = document.createElement('link');
-      link.type = 'text/css';
-      link.rel = 'stylesheet';
-      link.href = url;
+      
+      var present = false;
+      var link = undefined;
+      
+      for (var i = 0; i < links.length; i++) {
+        if (links[i].href.split('?')[0] === url) {
```

Comment:
I was working under the assumption that url would not have a query parameter, if that is not true then I can add the url as a data-attribute instead. If url can have a query parameter then I will also need to append the timestamp with '&' I think.